src/runtime/analytics-service.js

```diff
@@ -184,10 +184,10 @@ export class AnalyticsService {
     if (!this.serviceReady_) {
       // TODO(sohanirao): Potentially do this even earlier
       this.doc_.getBody().appendChild(this.getElement());
+      this.setContext_();
```

Comment:
Just curious, why did this move up?

Reply:
So the data put in context was there sooner (in case the activities are going to use it)